taxi/views.py

```diff
@@ -87,3 +106,25 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+    success_url = reverse_lazy("taxi:driver-list")
+    template_name = "taxi/driver_form.html"
```

Comment:
Code Efficiency: In the `DriverCreateView`, the form used is `DriverForm`. According to the checklist, while creating a user, you should be using `UserCreationForm`. Please update your `DriverForm` to inherit from `UserCreationForm` if it's not doing so already.